dnplab/dnpIO/parspc.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np
+from dnplab import dnpdata
+import warnings
+
+
+def import_parspc(path):
```

Comment:
I think this module should be named elexsys.

Reply:
elexsys use DTA/DSC format, this par/spc format is for winepr and ESP systems. But maybe you mean the bes3t module? Also, elexsys systems only use Xepr but the EMX series (and nano) that run Xenon are also bes3t. In the import wrapper you can specify 'xenon' or 'xepr' and both will route to the bes3t module, and 'winepr' or 'esp' both will route to the parspc module. Or, leave out the type altogether and it will select the correct module based on the file extension.

Reply:
Yeah, you're right, I should have said "winepr" instead. I'm just not a fan of the "parspc" name. I would prefer a name that is related to the name of the spectrometer software. I'm fine with the other module being called "bes3t"

Reply:
Ok I agree with that, I made the change to winepr and updated import wrapper and unittests accordingly. I also fixed a bunch more cases where os.path.splitext is a better option.